### PR TITLE
Adds plastic floors, floor tiles.

### PR DESCRIPTION
**Pull request by ArcaneMusic**
## About The Pull Request

With the new plumbing features, plastic is now a lot easier to make en-masse, so this gives an extra use for plastic, some cheap plastic flooring.

I'm open for suggestions on any changes to the sprites.
image (Most recent as of 10/26)

## Why It's Good For The Game

Realistically, plastic is a very useful material, and having a cheap replacement for full metal flooring fits the theme for plastic. It also gives chemistry another incentive to make plastic beyond just large plastic bottles.

## Changelog 
:cl:
add: Plastic can now be used to make cheap, plastic floor tiles.
/:cl:

### CODE REVIEW

**Comment by Jalleo** [comment]
Could you make them more shiny instead of prickled? Or instead remove the prickles? It would probably be a bit too distracting and ugly as is but without that would look really nice.

File: code/game/turfs/simulated/floor/misc_floor.dm [hunk 1]
```diff
@@ -163,3 +163,9 @@
 /turf/open/floor/black
 	name = "black floor"
 	icon_state = "black"
+
+/turf/open/floor/plastic
+	name = "plastic floor"
+	desc = "Cheap, lightweight flooring."
```
Comment (PKPenguin321) [inline code/game/turfs/simulated/floor/misc_floor.dm]: Would be cool if they were actually weaker to reflect this, like the inverse of reinforced plating

**Comment by ArcaneMusic** [comment]
image

Alright, I've tried a few different styles for the floor, for now it's closer to a traditional floor tile more than anything but with a bit more gloss and shine than regular metal flooring.

Also, I gave it a higher thermal conductivity and fairly low heat capacity, so that they're more likely to break when exposed to heat/damage.

**Comment by nicbn** [comment]
They're off-center and it hurts

File: code/game/turfs/simulated/floor/misc_floor.dm [hunk 2]
```diff
@@ -163,3 +163,12 @@
 /turf/open/floor/black
 	name = "black floor"
 	icon_state = "black"
+
+/turf/open/floor/plastic
```
Comment (Qustinnus) [inline code/game/turfs/simulated/floor/misc_floor.dm]: you need to add datum mats here too

Comment (ArcaneMusic) [reply code/game/turfs/simulated/floor/misc_floor.dm]: Done.

**Comment by carlarctg** [comment]
if you want to make plastic even more useful, someone suggested a plastic lawnchair that breaks if a fattie sits on it

**Comment by Cyprex** [comment]
The weakness to heat is nice, another unique property would also be great to differentiate these from other tiles.

File: code/game/objects/items/stacks/tiles/tile_types.dm [hunk 3]
```diff
@@ -326,3 +326,11 @@
 	custom_materials = null // All other Borg versions of items have no Metal or Glass - RR
 	is_cyborg = 1
 	cost = 125
+
+/obj/item/stack/tile/plastic
+	name = "plastic tile"
+	singular_name = "platic floor tile"
```
Comment (kriskog) [inline code/game/objects/items/stacks/tiles/tile_types.dm]: plastic

**Comment by actioninja** [comment]
why are the tiles not square

**Comment by ArcaneMusic** [comment]
All tile sprites are slightly rounded, and I fixed the turf sprites so the tiles are square again. The screenshot was from the day before, but it's corrected now.

**Comment by actioninja** [comment]
you best not be tryin to cheat me

**Pull request merged**